### PR TITLE
Finished with USGS Muhs

### PR DESCRIPTION
**Pull request by uthorat01**
Link this to issue #39

### CODE REVIEW

File: data_reports/111_USGSMuhs.Rmd [hunk 1]
```diff
@@ -0,0 +1,199 @@
+USGS Muhs
+
+In this section, we will pull out `USGS Muhs` from ISCN3 and clean it up. 
+Empty columns were removed from the dataset and missing values were filled in. This data set required a cryptic message in the profile of the dataset to be formally cited using BibTeX. (https://doi.org/10.1016/S0277-3791(03)00167-7)
```
Comment (ktoddbrown) [inline data_reports/111_USGSMuhs.Rmd]: Need a few more details here. Were there any ISCN gap filled values, if so how were those dealt with? Were there any duplicate data in the rows?

File: data_reports/111_USGSMuhs.Rmd [hunk 2]
```diff
@@ -0,0 +1,199 @@
+USGS Muhs
```
Comment (ktoddbrown) [inline data_reports/111_USGSMuhs.Rmd]: This needs a '#' so that USGS Muhs is a first level header.

File: data_reports/111_USGSMuhs.Rmd (hunk 1 above)
Comment (ktoddbrown) [reply data_reports/111_USGSMuhs.Rmd]: Also need citation using the @ format up here instead of giving the doi.

**Review by ktoddbrown — CHANGES_REQUESTED** [review]
Close! need some more comments on how the dataset was processed.

File: data_reports/111_USGSMuhs.Rmd (hunk 1 above)
Comment (uthorat01) [reply data_reports/111_USGSMuhs.Rmd]: Fixed

File: data_reports/111_USGSMuhs.Rmd (hunk 2 above)
Comment (uthorat01) [reply data_reports/111_USGSMuhs.Rmd]: Fixed

**Review by ktoddbrown — APPROVED** [review]

**Pull request merged**